client/app/src/main/java/com/microsoft/research/karya/ui/registration/RegistrationActivity.kt

```diff
@@ -4,8 +4,10 @@ import android.os.Bundle
 import android.util.Log
 import com.microsoft.research.karya.R
 import com.microsoft.research.karya.ui.base.BaseActivity
+import dagger.hilt.android.AndroidEntryPoint
 
-class RegistrationActivity : BaseActivity(useAssistant = true) {
+@AndroidEntryPoint
+class RegistrationActivity : BaseActivity() {
```

Comment:
Please use `AppCompatActivity` instead of `BaseActivity`

Reply:
Yes, Would do it as soon as BaseActivity is purged. BaseActivity has containers which is used by RegistrationActivity. I cant remove it from current implmentation